lib/grape/validations/validators/exactly_one_of.rb

```diff
@@ -7,7 +7,8 @@ module Validations
     class ExactlyOneOfValidator < MultipleParamsBase
       def validate_params!(params)
         return if keys_in_common(params).length == 1
-        raise Grape::Exceptions::Validation.new(params: all_keys, message: message(:exactly_one))
+        raise Grape::Exceptions::Validation.new(params: all_keys, message: message(:exactly_one)) if keys_in_common(params).length.zero?
```

Comment:
We do a lot of `keys_in_common` calls, let's fix this?

```ruby
_keys_in_common = keys_in_common(params)
return if _keys_in_common.length == 1
raise ... if _keys_in_common.none?
raise 
```

Reply:
of course